compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/v2/Scrollbar.desktop.kt

```diff
@@ -335,6 +337,26 @@ internal class LazyGridScrollbarAdapter(
 }
 
 
+@ExperimentalFoundationApi
+internal class TextFieldScrollbarAdapter(
```

Comment:
this is an internal class, so we don't need annotation ExperimentalFoundationApi

Reply:
Right, I'll change to `@OptIn`